WordPress/WordPressUITests/Screens/Me/SupportScreen.swift

```diff
@@ -0,0 +1,45 @@
+import UITestsFoundation
+import XCTest
+
+// This screen object is for the Support section. In the app, it's a modal we can get to from Me > Help & Support, or, when logged out, from Prologue > tap either continue button > Help.
```

Comment:
Nitpick / Tip. You could make this a documentation comment with `///` instead of `//` and move it above the `SupportScreen` definition. Doing so would allow Xcode to generate an helper tooltip with that information.

This format:

![image](https://user-images.githubusercontent.com/1218433/125238488-a6b5dc80-e32a-11eb-904d-9ae21d6f1d23.png)

Then becomes this in the autocompletion window (see bottom of the screenshot)

![image](https://user-images.githubusercontent.com/1218433/125238522-b0d7db00-e32a-11eb-9580-5fee15d2d8f7.png)

And this if you `Option Click` on the type name:

![image](https://user-images.githubusercontent.com/1218433/125238605-cc42e600-e32a-11eb-8703-833074f0d0fa.png)

It's a minor thing, but in the optic of enabling developer to write these tests for themselves as part of the standard feature work, having documentation like this handy is an advantage.

Reply:
Oooh I love this, great tip! Implemented, thanks!